.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
Well, when we move to plugins, it should reduce the size of our directory hopefully pretty significantly due to all the vendored code.  But, perhaps we should move forward with this as-is and deal with that if/when it is a problem.